lib/screens/devices/device_list_screen.dart

```diff
@@ -40,39 +41,29 @@ class _DeviceListScreenState extends State<DeviceListScreen> with LogMixin {
 
   @override
   Widget build(BuildContext context) {
-    final consumerWidget = Consumer2<ChangeCallbackNotifier<PeerInfoService>,
-        ChangeCallbackNotifier<PeerService>>(
-      builder: (context, service, peerService, child) => _buildDeviceList(
-        context,
-        service.callbackProvider,
-        peerService.callbackProvider,
-      ),
+    final showQrScannerButton = _showQrScannerButton();
+    final consumerWidget = Consumer2<PeerInfoService, PeerService>(
```

Comment:
Shouldn't it be `ChangeCallbackNotifier<PeerInfoService>` and `ChangeCallbackNotifier<PeerService>`?